drivers/vulkan/rendering_device_vulkan.h

```diff
@@ -546,7 +546,7 @@ class RenderingDeviceVulkan : public RenderingDevice {
 	};
 
 	struct UniformInfo {
-		UniformType type = UniformType::UNIFORM_TYPE_MAX;
+		ParameterType type = ParameterType::PARAMETER_TYPE_MAX;
```

Comment:
I don't think we should change the RenderingDevice API. Its better to keep this closer to the actual names. It won't be used by beginners either, i.e. you have to know what a uniform is in order to use the RD API